lib/web_ui/dev/felt.dart

```diff
@@ -38,7 +38,11 @@ void main(List<String> rawArgs) async {
 
   int exitCode = -1;
   try {
-    final bool result = (await runner.run(args)) as bool;
+    final bool? result = await runner.run(args);
+    if (result == null) {
+      // result == null means Usage was printed.
+      exitCode = 0;
```

Comment:
IIRC the convention is for usage to exit with code 64. It would also protect from accidental nulls. We'd want the CI to fail when that happens, which won't happen if we exit with 0.